### ISSUE TITLE
Local backups with Album-like folder structure

### ISSUE THREAD

**Issue by pluja** [photoprism/photoprism#620]
I'd love a feature that allowed to either manually or automatically backup all media to a local backup drive of choice conserving an Album-like folder structure.

So for example one could have the following backet-up folder sctructure:
```
Photos/
├── 2020/
│   ├── Holiday with family
│   ├── Trip to Mount Kilimanjaro
│   ├── 11-November*
│   └── Night with friends
├── 2019/
│   ├── Christmas
│   └── 09-September*
```

Basically what I mean is that when backing up we could preserve a folder structure that mimics the organization that the user has on Photoprism. This is because I'd like to, If I ever remove Photoprism or anything like that, be able to see my organized photos on any computer just with the folder structure.

The ones marked with `*` would contain any photos that have no album assigned, so they go the the `Year/Month` they correspond.

**Comment by lastzero** [comment]
I remember a similar feature request, maybe you can find it. Main issue here is that not all photos are in an album and some are in multiple albums. This also changes over time. So it's really just a snapshot for exports, not something you can use for proper backups (where you only want to sync / upload changes and no duplicates).

**Comment by pluja** [comment]
True. Maybe ignore all albums and use a Date-based folder structure. Just like this:

```
Photos/
├── 2020/
│   ├── 01-January
│   ├── 02-February
│   ├── 11-November
│   └── 12-December
└── 2019/
     ├── 08-August
     └── 09-September
```
IMO this would be very convenient when backing-up photos, thinking in a no-photoprism-available system.

I think the issue you mention is #327
